execution/wasm/contract.go

```diff
@@ -84,6 +85,12 @@ func (ctx *context) ResolveFunc(module, field string) lifeExec.FunctionImport {
 
 	switch field {
 	case "call":
+		fallthrough
+	case "callCode":
+		fallthrough
+	case "callDelegate":
+		fallthrough
+	case "callStatic":
```

Comment:
you can comma-delimiter these values under a single case statement, think that is cleaner/more/less likely to accidentally split cases.

Reply:
right, that is nicer. Something for the next ewasm pr